packages/launcher/src/index.tsx

```diff
@@ -303,14 +303,14 @@ class Launcher extends VDomRenderer<LauncherModel> {
       let kernel = KERNEL_CATEGORIES.indexOf(cat) > -1;
       if (cat in categories) {
         section = (
-          <div className='jp-Launcher-section'>
+          <div className='jp-Launcher-section' key={index}>
```

Comment:
If the category is known and unique, it's a better key than the `index`. This matters if the ordering of the elements could change or if any are added or deleted.

Reply:
Updated, thanks.  It looks like we can't do the same for the `Card`s because they could differ only by callback.

Reply:
I stand corrected, we can and should enforce uniqueness using a stable `JSON.stringify`.